packages/table/src/index.ts

```diff
@@ -16,41 +16,41 @@
 
 /* eslint-disable deprecation/deprecation */
 
-export { Cell, CellProps, ICellProps, ICellRenderer, CellRenderer } from "./cell/cell";
+export { Cell, type CellProps, type ICellProps, type ICellRenderer, type CellRenderer } from "./cell/cell";
```

Comment:
I was getting a bunch of unrelated webpack errors with these exports, and adding these `type` annotations seems to fix them:

<img width="1501" alt="Screenshot 2023-02-27 at 3 15 59 PM" src="https://user-images.githubusercontent.com/723999/221679730-f7d1592a-a977-4e32-9925-00909abaef9a.png">